src/components/GridLayout.js

```diff
@@ -54,6 +55,28 @@ class GridLayout extends Component {
   componentWillMount() {
     const { columnCount } = this.props;
     if (!this.state.layout) {
+      let layout;
+      if (this.props.gridSizeIsConfigurablePerTile === true) {
+        // use layout as configured on tile in client_configuaration json
+        layout = this.props.tiles.map((tile, i) => {
+          return tile.sizeAndLocaltionInGrid;
```

Comment:
En een => functie die alleen iets returnt kan beter zonder de { }, dus het kan gewoon als `tile => tile.sizeAndLocationInGrid` volgens mij.

Maar je hebt hier Localtion staan, typo! Wel getest?

Reply:
Overigens zeggen de docs boven dat het niet een property .sizeAndLocationInGrid moet zijn, maar dat ze op root level op de tile moeten zitten. Wat is het?